web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/ytelsefordeling/OmsorgOgRettDto.java

```diff
@@ -0,0 +1,29 @@
+package no.nav.foreldrepenger.web.app.tjenester.behandling.ytelsefordeling;
+
+import java.util.Set;
+
+import no.nav.foreldrepenger.behandlingslager.behandling.personopplysning.SivilstandType;
+import no.nav.foreldrepenger.behandlingslager.geografisk.Landkoder;
+import no.nav.foreldrepenger.web.app.tjenester.behandling.personopplysning.PersonadresseDto;
+
+public record OmsorgOgRettDto(Søknad søknad, RegisterData registerdata, ManuellBehandlingResultat manuellBehandlingResultat) {
+
+    public record Søknad(Boolean søkerHarAleneomsorg, String annenpartNavn, String annenpartIdent, Landkoder annenpartBostedsland,
+                         Rettighet annenpartRettighet) {
+
+        public record Rettighet(boolean harRettNorge, Boolean harOppholdEØS, Boolean harRettEØS, Boolean harUføretrygd) {
```

Comment:
Er det eksisterende aksjonspunkt-logikk som driver dette - eller behovet for å vise hva som er valgt i søknad + aksjonspunkt?

Alternativet ville vært å hatt en enum som viser Rolle+Rettighet - BEGGE, BEGGEEØS, R_ALENE, R_ENERETT

Reply:
Nå fokuserer vi på visning av søknad + registerdata + saksbehandlet (dagens). 
Det kan tenkes at manuell behandling delen endrer seg hvis vi senere skal endre på selve aksjonspunktet resultatet. Da vil søknad fortsatt inneholde det samme (Rettighet), men ManuellBehandlingResultat ev inneholde en enum